src/it/scala/djinni/MetadataIntegrationTest.scala

```diff
@@ -0,0 +1,19 @@
+package djinni
+
+import org.scalatest.GivenWhenThen
+import org.scalatest.Matchers.{convertToAnyShouldWrapper, startWith, include}
+
+class MetadataIntegrationTest extends IntegrationTest with GivenWhenThen {
+  it("should output version information") {
+    When("calling `djinni --version`")
+    val versionDescription = djinni("--version")
+    Then(s"the version description should be returned")
+    versionDescription should startWith ("djinni generator version")
```

Comment:
This is not an amazing test. All I do is test if the cli output starts with the correct string. I do not verify that a properly formatted version is appended.

The test at least ensures the program won't crash without outputting anything.
One could in the future add a regex evaluation of the version to get even more confidence.

Reply:
this test is fine, I can add an additional test in the conan recipe that I have, here I currently also just test that djinni returns something, but I can adopt that to call --version and parse the output, once the next version is tagged